cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -371,3 +394,533 @@ func TestMachinesInMachineSet(t *testing.T) {
 		}
 	}
 }
+
+func TestControllerNodeGroupsSizes(t *testing.T) {
+	for i, tc := range []struct {
+		description string
+		annotations map[string]string
+		count       int
+	}{{
+		description: "errors because minSize is invalid",
+		annotations: map[string]string{
+			nodeGroupMinSizeAnnotationKey: "-1",
+			nodeGroupMaxSizeAnnotationKey: "0",
+		},
+	}, {
+		description: "errors because maxSize is invalid",
+		annotations: map[string]string{
+			nodeGroupMinSizeAnnotationKey: "0",
+			nodeGroupMaxSizeAnnotationKey: "-1",
+		},
+	}, {
+		description: "errors because minSize > maxSize",
+		annotations: map[string]string{
+			nodeGroupMinSizeAnnotationKey: "1",
+			nodeGroupMaxSizeAnnotationKey: "0",
+		},
+	}, {
+		description: "errors because maxSize < minSize",
+		annotations: map[string]string{
+			nodeGroupMinSizeAnnotationKey: "1",
+			nodeGroupMaxSizeAnnotationKey: "0",
+		},
+	}, {
+		annotations: map[string]string{
```

Comment:
`description` here?

Reply:
Addressed in https://github.com/openshift/kubernetes-autoscaler/pull/34/commits/6b96d42b64166a3ca2abf8436f8a427269230be4